### PR TITLE
nodebrew 0.9.2

### PR DESCRIPTION
**Pull request by lion-man44**
Hi.

Update nodebrew 0.9.2 version

### CODE REVIEW

File: Library/Formula/nodebrew.rb [hunk 1]
```diff
@@ -22,6 +22,6 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert shell_output("#{bin}/nodebrew ls-remote").include?("v0.10.0")
+    assert_match version.to_s, shell_output("#{bin}/nodebrew --version | egrep \"nodebrew \\d\"")
```
Comment (bfontaine) [inline Library/Formula/nodebrew.rb]: This ` | egrep \"nodebrew \\d\"` shouldn’t be necessary here.

Comment (xu-cheng) [inline Library/Formula/nodebrew.rb]: Also, why replace the test? The old test `nodebrew ls-remote` is preferred than simple `--version` test.

Comment (DomT4) [inline Library/Formula/nodebrew.rb]: Maybe the audit complained about `assert...include`?

```ruby
assert_match /v0.10.0/, shell_output("#{bin}/nodebrew ls-remote")
```
Should work on that though.

Comment (lion-man44) [inline Library/Formula/nodebrew.rb]: @DomT4 
> Maybe the audit complained about assert...include?

Yes.

> Should work on that though.

It's good.
But, what is `v0.10.0`?

Comment (DomT4) [inline Library/Formula/nodebrew.rb]: > But, what is v0.10.0?

Don't use Nodebrew personally but the naming of `ls-remote` would suggest its purpose is to list all the Nodes available for installation, and back in 2013 or so there was a Node `v0.10.0` release which this presumably matches.

Comment (lion-man44) [inline Library/Formula/nodebrew.rb]: Oh, sorry.
Typo what -> why.

Thanks for advice.
Code is fixed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/f75a13fa099200501c2a13c5e01180e5120682f4. Thank you for your contribution to Homebrew @lion-man44; we appreciate it! :smiley_cat:

**Pull request closed without merging**